_posts/2016-10-10-making-tsuru-easier-to-install.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+status: publish
+published: true
+title: Making tsuru easier to install on any cloud
+author:
+  display_name: André Carvalho
+  url: https://github.com/andrestc
+date: '2016-10-10 15:05:00 -0300'
+date_gmt: '2016-10-10 18:05:00'
+categories:
+- Features
+tags:
+- docker machine
+- swarm
+- docker
+comments: []
+---
+
+tsuru client 1.1 comes with a experimental feature called tsuru Installer. With tsuru Installer
+it is easy to start a new tsuru installation, including a multi-node installation. It's an great
+way to try tsuru out, both on a local machine or in a multi-node installation on the cloud.
+
+To install tsuru locally(in a virtualbox VM), just run:
+
+`tsuru install`
+
+It's possible to use the installer to provision an entire multi-node infrastructure to run tsuru on a
+large set of IaaS, like EC2, Azure, GCE etc. For more information, check the [documentation](https://docs.tsuru.io/stable/experimental/installer.html).
+
+How It Works
+------------
+
+tsuru installer leverages [docker machine](https://docs.docker.com/machine/) to create docker hosts and that is what makes it able to provision hosts on a large set of IaaS and also makes it extensible: if your IaaS is not supported by docker machine, it is easy to develop an driver to do the heavy lifting.
```

Comment:
should be `Docker Machine` instead `docker machine` and `Docker` instead `docker`